### ISSUE TITLE
Town Map Icon Bug

### ISSUE THREAD

**Issue by MAGNAT2645** [mrtenda/CoilSnake#173]
Red "Hint" icon just won't appear on any Town Map. YML icon position (**only Hint icon**) looks fine:
```
0:
- Event Flag: 0x82a9
  Icon: hint
  X: 187
  Y: 94
1:
- Event Flag: 0x82aa
  Icon: hint
  X: 208
  Y: 61
2:
- Event Flag: 0x82ab
  Icon: hint
  X: 215
  Y: 10
3:
- Event Flag: 0x82ac
  Icon: hint
  X: 88
  Y: 119
4:
- Event Flag: 0x82ae
  Icon: hint
  X: 141
  Y: 94
5:
- Event Flag: 0x82ad
  Icon: hint
  X: 72
  Y: 108
```

This is what my icons image looks like:
![icons](https://user-images.githubusercontent.com/11666918/51673574-22533000-1ff0-11e9-9611-e74c70c719d1.png)

**Comment by MAGNAT2645** [comment]
It's same as Vanilla Earthbound. Also i don't know how to check it properly while testing.

**Comment by mrtenda** [comment]
The icons don't appear normally in EB either, so this is the expected behavior. I think they only appear after you enter the hint man's house. I suggest testing it out in unhacked EB to understand the behavior.

**Comment by MAGNAT2645** [comment]
But hint icon appears in Vanilla Earthbound (ex. Threed Map) even if you're outside.

**Comment by mrtenda** [comment]
Yes. If you ever walk inside the hint man's house once, then the icon will always appear, even after you leave the house. I think this is how it works in EarthBound.
